tests/StoringGroceries.tex

```diff
@@ -47,7 +47,7 @@ \subsection{Task}
 	\begin{itemize}
 		\item \textit{Inspect the cupboard} (locating and categorizing existing groceries).
 		\item \textit{Open the cupboard's door.} If the robot can't open the door, it may ask the Referee to do it.
-		\item \textit{Find the table}
+		\item \textit{Find the table} Only a rough location is defined beforehand
```

Comment:
This is unnecessary. Instead let's add **Table** under Additional Rules and Remarks.

_The table location will be roughly announced beforehand, having it's position either left, right, or behind the robot. The distance between the cupboard and the table will be adjusted to prevent robot from displacing (i.e. only turning)._

Reply:
Adjusting the table position is also not needed I think. If the arena puts a table close enough to a closest in the default arena state, I'd definitely would like to use that table.